src/AddTodoForm.jsx

```diff
@@ -23,14 +24,9 @@ function AddTodoForm({ onAddTodo }) {
     return (
         <div>
             <form onSubmit={handleAddTodo}>
-                <label htmlFor="todoTitle">Title </label>
-                <input
-                    id="todoTitle"
-                    type="text"
-                    name="title"
-                    value={todoTitle}
-                    onChange={handleTitleChange}
-                />
+                <InputWithLabel value={todoTitle} onChange={handleTitleChange} autoFocus>
```

Comment:
The autoFocus isn't needed because the InputWithLabel component already handles focus internally with useRef.

Reply:
fixed